src/wfs/src/main/java/org/geoserver/wfs/json/GeoJSONGetFeatureResponse.java

```diff
@@ -290,6 +225,123 @@ else if ( id_option.length() != 0){
         }
     }
 
+    public class FeaturesInfo {
```

Comment:
So let's make it private ! 
Jokes apart this class should be private and I will add some Java doc anyway, thanks.